### PR TITLE
Test files for statement_annotations

### PR DESCRIPTION
**Pull request by odscrachel**
# Overview

- What does this pull request do? Creates valid and invalid test files for annotations 
- How can a reviewer test or examine your changes?
- Who is best placed to review it?

Closes #

## Translations

- [ ] New or edited strings appearing as a result of this PR will be picked up for translation
- [ ] I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

- [ ] I've checked that the documentation builds without failing. See: https://github.com/openownership/data-standard#building-the-documentation-locally
- [ ] I've thought about how and when this needs to be released. See:
      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- [ ] I've updated the changelog, if necessary.
- [ ] I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- [ ] I've added or edited any other related documentation. See: https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md

### CODE REVIEW

File: tests/data/valid-statements/statement_annotations_transformedContent.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "statementId": "2f7bf9370f1254068e5e946df067d07d",
+        "declarationSubject": "xyz",
+        "statementDate": "2017-11-18",
+        "annotations": [
+            {
+                "statementPointerTarget": "/recordDetails/interestedParty",
```
Comment (kathryn-ods) [inline tests/data/valid-statements/statement_annotations_transformedContent.json:8]: not sure this makes sense with this being an entity statement, it doesn't have this field?

**Review by kathryn-ods — COMMENTED** [review]
mostly all good - one file with notes

File: tests/data/valid-statements/statement_annotations_transformedContent.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                "motivation": "transformation",
+                "transformedContent": "12-31-2020"
```
[the remark this answers is not in the document]
Comment (odscrachel) [reply tests/data/valid-statements/statement_annotations_transformedContent.json:10]: Great, I'll do that

**Comment by odscrachel** [comment]
Thanks Kathryn, I've updated that file to point to the statementDate, I also updated the invalid file so that it makes sense for an entity statement.

**Review by kathryn-ods — APPROVED** [review]

**Pull request merged**